### ISSUE TITLE
Account activities page: /@demo/activities

### ISSUE THREAD

**Issue by feruzm** [ecency/ecency-vision#1058]
Where we show latest activities, like votes, replies and posts, etc.

**Comment by igormuba** [comment]
@feruzm should this be a new tab on the profile page? What I understood is that this page will compile all activity that otherwise would be separate, so you could see only posts, only reblogs, only votes (to be done curation page) or this other activities page with everything in one place, right?

**Comment by feruzm** [comment]
@igormuba Yes, We will have Activities tab and will include curation trail within that page as well. Basically more visualized version of block explorer. So while this is #1157 separate page with its own URL but component we use on that page will basically be part of Activities page as well.
For example, design elements are iterated but use your creativity here...

- votes would be presented also like feed but indicator that it has been voted by that user (trail components #1157 used on this type of activity)
- replies also parent post with reply content/link is represented more visually appeal way improvement on block explorer style view.
- comments
- reblogs,
- posts, etc. similar pattern
- content rewards could also be part of it.

This will give us option to remove Replies and Comments from profile tabs (while keeping dedicated URL intact /@demo/replies, /@demo/comments) and add them inside Activities tab.

Let me know if you need more clarifications.
